app/models/comment.rb

```diff
@@ -1,6 +1,9 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :post
+  has_many   :subcomments, class_name: 'Comment', foreign_key: :parent_id,
```

Comment:
czy na pewno tak powinno byc, ze jak usuniesz main comment to subkomenty tez sa usuwane? (jesli dobrze rozumiem twój kod)

mi się wydaje że powinniśmy zostawić parent koment w UI, i ewentualnie zaznaczyć że [comment deleted by admin] a odpowiedzi chyba warto zostawić. Ale sam nie wiem